src/mapping.ts

```diff
@@ -57,6 +57,8 @@ export function handleTransfer(event: Transfer): void {
   talentToken.txCount = talentToken.txCount.plus(ONE_BI);
 
   talentToken.save()
+
+  updateTalentDayData(event)
```

Comment:
Should we also call this when a token is created and set the initial supply to 2000 TAL? How are we managing this initial supply?

Reply:
Yup, you're right. By default when the mint method is called the Transfer event is dispatched.